clang/tools/dpct/DpctOptRules/cmake_script_migration_rule_optional.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) Intel Corporation
```

Comment:
Why add a new file "clang/tools/dpct/DpctOptRules/cmake_script_migration_rule_optional.yaml"?  It is auto-loaded by SYCLomatic.

Reply:
Yes, this the reason why the PR add the rule into a sperate file.

Reply:
Okay